File: Shopping Cart/Shopping Cart - CSharp/Form1.cs
using System;
using System.Windows.Forms;

namespace Shopping_Cart
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            int total = 0;
            if (CheckBox1.Checked)
            {
                total += 50000;
            }
            if (CheckBox2.Checked)
            {
                total += 10000;
            }
            if (CheckBox3.Checked)
            {
                total += 8000;
            }

            TextBox1.Text = "Total is : ₹ " + total.ToString();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            CheckBox1.Checked = false;
            CheckBox2.Checked = false;
            CheckBox3.Checked = false;
            TextBox1.Text = "Total is :";
        }
    }
}

File: Shopping Cart/Shopping Cart/Form1.vb
Public Class Form1
    Private ReadOnly _cartService As New Shopping_Cart.ShoppingCartService()

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim total As Integer = _cartService.CalculateTotal(
            CheckBox1.Checked,
            CheckBox2.Checked,
            CheckBox3.Checked
        )
        TextBox1.Text = _cartService.FormatTotal(total)
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        CheckBox1.Checked = False
        CheckBox2.Checked = False
        CheckBox3.Checked = False
        TextBox1.Text = "Total is :"
    End Sub
End Class
